### DISCUSSION TITLE
GitHub Copilot插件崩溃

### DISCUSSION THREAD

**Discussion by wmuj** [community/community#143603] [Copilot Conversations]
### Select Topic Area

Question

### Body

<img width="980" alt="7c0db6832f1042930082e63c9561845" src="https://github.com/user-attachments/assets/7e504d8d-3070-499b-ba15-2f00cc80a8f2">
今天更新完IDEA，运行插件时候不出现代码提示，登录完，添加注释没有代码提示而且底部图标显示灰色，登录完又重复一遍，真的是体验感非常差！！！
![image](https://github.com/user-attachments/assets/3417690b-e98c-470e-9cbd-7f9ab24d684f)

**Comment by wmuj** [comment]
没有解决！！！1

**Comment by IasonNN** [comment]
Same problem here.

**Comment by Akash1134** [comment]
Hi @wmuj , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
